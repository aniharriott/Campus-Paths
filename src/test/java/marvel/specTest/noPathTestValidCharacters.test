# Find no path between valid characters using noPath.

LoadGraph noPath noPath.tsv

FindPath noPath a d