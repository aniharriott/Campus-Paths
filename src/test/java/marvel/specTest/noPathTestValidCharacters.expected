# Find no path between valid characters using noPath.

loaded graph noPath

path from a to d:
no path found